Add required PMC policies (#1106)
openjournals/joss: docs/policies.md

JOSS Policies
==========================

## Animal research policy

> In the exceptional case a JOSS submission contains original data on animal research, the corresponding author must confirm that the data was collected in accordance with the latest guidelines and applicable regulations. The manuscript must include complete reporting of the data, including the ways that the study was approved and relevant details of the sample. Authors are required to report either the [ARRIVE](https://arriveguidelines.org/) or PREPARE guidelines for animal research in the submission. 

Reviewer: Is there a link for PREPARE as well?

Author: Woopsie! Adding!

We recommend authors to replace, reduce, and refine animal research as promoted by the [N3RS](https://www.nc3rs.org.uk/).

## Data sharing policy

> If any original data analysis results are provided within the manuscript, this analysis has to be entirely reproducible by reviewers, including accessing the data. Submissions are by definition contained within a repository and we require authors to archive their software before we accept.

Reviewer: ```suggestion
If any original data analysis results are provided within the submitted work, such results have to be entirely reproducible by reviewers, including accessing the data. Submissions are, by definition, contained within a repository and we require authors to archive their software before we accept.
```

Reviewer: I don't quite understand the second sentence here.

Submissions are typically a single repo but not always.

And how does requiring that authors archive their software fit with the first sentence about data analyses?

Author: Thanks @danielskatz!

Would this work?

> If any original data analysis results are provided within the submitted work, such results have to be entirely reproducible by reviewers, including accessing the data. 

> Submissions are, by definition, contained within one or multiple repositories, which we require authors to archive before we accept the submission. Any data contained within the software is made available accordingly.

A slight adaptation of your suggestion 😊
